web-wallet/src/routes/(app)/settings/__tests__/page.spec.js

```diff
@@ -75,6 +88,21 @@ describe("Settings", () => {
 		expect(resetButton).toHaveAttribute("data-tooltip-disabled", "false");
 	});
 
+	it("should disable the \"Back\" button if invalid gas limit or price are introduced", async () => {
+		const { getByLabelText, getByRole } = render(Settings, {});
+		const priceInput = asInput(getByLabelText(/price/i));
+		const limitInput = asInput(getByLabelText(/limit/i));
+		const backButton = getByRole("link");
+
+		await fireInput(priceInput, 30000000);
+		expect(backButton).toHaveAttribute("aria-disabled", "true");
+		await fireInput(priceInput, 20000000);
+
+		await fireInput(limitInput, 3000000000);
+		expect(backButton).toHaveAttribute("aria-disabled", "true");
+		await fireInput(limitInput, 20000000);
```

Comment:
Aren't you missing an expectation after this?

Reply:
Same as above

Reply:
Same as above as well!